docker-compose.yaml

```diff
@@ -10,6 +10,8 @@ services:
       - DEBUG=True
       - DATABASE_URL=postgresql://admin:admin_pw@database/admin
       - FEEDBACK_EMAIL=office-hours-devs@umich.edu
```

Comment:
As suggested in earlier comments, let's move these to `.env`, too.

Reply:
Sure we can move these, but I think having these might *require* that a .env file exists unless these are in the code. That's probably fine as a practice. I'd also update the README, I was still testing this to make sure there was nothing else and having it as draft.

This would only affect development but is really useful to have for development.